src/explore-education-statistics-common/src/modules/table-tool/contexts/TableHeadersContext.tsx

```diff
@@ -0,0 +1,109 @@
+import useToggle from '@common/hooks/useToggle';
+import { noop } from 'lodash';
```

Comment:
Not too certain but should we import noop like `import noop from 'lodash/noop';` to avoid importing the entire library here?